### PR TITLE
Translate providerName

### PR DESCRIPTION
**Pull request by VincentLanglet**
Follow up of https://github.com/sonata-project/SonataMediaBundle/pull/2291

### CODE REVIEW

File: src/Resources/views/MediaAdmin/select_provider.html.twig [hunk 1]
```diff
@@ -27,15 +27,15 @@ file that was distributed with this source code.
                        class="btn btn-app btn-block"
                        data-toggle="tooltip"
                        data-placement="top"
-                       title="{{ provider.providerMetadata.description|trans({}, provider.providerMetadata.domain|default('SonataMediaBundle')) }}"
+                       title="{{ provider.providerMetadata.description|trans({}, provider.providerMetadata.domain|default(admin.getTranslationDomain())) }}"
```
Comment (jordisala1991) [inline src/Resources/views/MediaAdmin/select_provider.html.twig]:
```suggestion
                       title="{{ provider.providerMetadata.description|trans({}, provider.providerMetadata.domain|default(admin.translationDomain)) }}"
```

File: src/Resources/views/MediaAdmin/select_provider.html.twig [hunk 2]
```diff
@@ -27,15 +27,15 @@ file that was distributed with this source code.
...
                             >
                         {% if provider.providerMetadata.option('class') %}
                             <i class="{{ provider.providerMetadata.option('class') }}"></i>
                         {% else %}
                             <img src="{{ asset(provider.providerMetadata.image) }}" style="max-height: 20px; max-width: 100px;"/>
                             <br/>
                         {% endif %}
-                        {{ provider.providerMetadata.title|trans({}, provider.providerMetadata.domain|default('SonataMediaBundle')) }}
+                        {{ provider.providerMetadata.title|trans({}, provider.providerMetadata.domain|default(admin.getTranslationDomain())) }}
```
Comment (jordisala1991) [inline src/Resources/views/MediaAdmin/select_provider.html.twig]:
```suggestion
                        {{ provider.providerMetadata.title|trans({}, provider.providerMetadata.domain|default(admin.translationDomain)) }}
```

**Review by jordisala1991 — APPROVED** [review]

**Comment by jordisala1991** [comment]
Thanks!

**Comment by VincentLanglet** [comment]
Do you want a release or do you want to add more changes ? @jordisala1991

**Comment by jordisala1991** [comment]
No, its fine, can you do a release?

**Pull request merged**